pkg/cfg/config.go

```diff
@@ -620,7 +619,8 @@ func (c *config) unmarshalStruct(key string, output interface{}, additionalDefau
 		settings, err := c.settings.Get(key).Map()
 
 		if err != nil {
-			c.err(fmt.Errorf("value is not of type map[string]interface{}: %w", err), "can not get settings for key: %s", key)
+			err = fmt.Errorf("value is not of type map[string]interface{}: %w", err)
```

Comment:
Is this line correct? What is `value` in this context?

Reply:
that wrapping can be dropped, as the `Map()` call already returns an appropriate error